app/src/test/java/io/appium/uiautomator2/model/settings/AbstractSettingTests.java

```diff
@@ -25,52 +25,94 @@
 
 import io.appium.uiautomator2.common.exceptions.UiAutomator2Exception;
 
+import static org.mockito.ArgumentMatchers.anyInt;
 import static org.mockito.ArgumentMatchers.anyLong;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.verify;
 import static org.mockito.MockitoAnnotations.initMocks;
 
 public class AbstractSettingTests {
-
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
     @Spy
-    private DummySetting dummySetting;
+    private DummyIntegerSetting dummyIntegerSetting;
+
+    @Spy
+    private DummyLongSetting dummyLongSetting;
 
     @Before
     public void setUp() {
         initMocks(this);
     }
 
     @Test
-    public void shouldThrowExceptionIfTypeIsNotValid() {
+    public void dummyIntegerSettingShouldThrowExceptionIfTypeIsNotValid() {
         thrown.expect(UiAutomator2Exception.class);
-        thrown.expectMessage("Invalid setting value type. Got: java.lang.String. Expected: java.lang.Long");
-        dummySetting.update("test");
+        thrown.expectMessage("Invalid setting value type. Got: java.lang.String. Expected: java.lang.Integer");
+        dummyIntegerSetting.update("test");
+    }
+
+    @Test
+    public void dummyIntegerSettingShouldNotThrowExceptionIfApplyFailed() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=5553523)